style.css

```diff
@@ -1,123 +1,377 @@
 @import url('https://fonts.googleapis.com/css2?family=Poppins:wght@300;500&display=swap');
 
+*,
+*::after,
+*::before {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
 .toolbar {
   display: flex;
   justify-content: space-between;
-  padding-left: 16px;
-  padding-right: 16px;
-  margin-bottom: 30px;
+  margin-top: 30px;
 }
 
 .logo {
-  width: 76px;
-  height: 20px;
+  width: 70%;
+  height: auto;
   font-family: 'Poppins', sans-serif;
   font-style: normal;
   font-weight: 700;
   font-size: 18px;
   line-height: 20px;
+  color: #6070ff;
+}
 
-  /* identical to box height, or 111% */
+a {
+  color: #6070ff;
+  text-decoration: none;
+}
 
+.first-page {
+  background-image: url('images/background.svg');
+  background-size: cover;
+  background-repeat: no-repeat;
+  width: 100%;
+  height: auto;
   display: flex;
-  align-items: center;
-  text-align: center;
+  justify-content: center;
+  flex-direction: column;
+}
 
-  /* Primary/B400 */
+.first-page .div-header {
+  width: 90%;
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  margin: 0 auto;
+  height: 100vh;
+}
 
-  color: #6070ff;
-  flex: none;
-  order: 0;
-  flex-grow: 0;
+.div-header .header {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  color: #172b4d;
+  width: 70%;
+  padding-bottom: 1rem;
+}
+
+.div-header .about-par {
+  width: 90%;
+  height: auto;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+  padding-bottom: 1rem;
+}
+
+.div-header .connect-page {
+  font-style: normal;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+  color: #7f8cff;
+  font-family: 'Poppins', sans-serif;
+  padding-bottom: 1rem;
+}
+
+.div-header .list {
+  display: flex;
+  gap: 1em;
+  justify-content: flex-start;
+  list-style-type: none;
+}
+
+.work-section {
+  background: #e5e5e5;
+  width: 100%;
+}
```

Comment:
- [ ] Please from the project requirement, you should use the grid to lay out this section.